crates/core/tedge_actors/src/builders.rs

```diff
@@ -330,8 +337,8 @@ pub trait RuntimeRequestSink {
 /// // Connect a test box to an actor under test
 /// let mut my_actor_builder = MyActorBuilder::new(MyActorConfig::default());
 /// let mut test_box_builder = SimpleMessageBoxBuilder::new("Test box", 16);
-/// my_actor_builder.register_peer(NoConfig, test_box_builder.get_sender());
-/// test_box_builder.register_peer(NoConfig, my_actor_builder.get_sender());
+/// my_actor_builder.connect_sink(NoConfig, &test_box_builder);
+/// my_actor_builder.connect_source(NoConfig, &mut test_box_builder);
```

Comment:
Now the `connect_sink` and  `connect_source` method are symmetric.

This could have been written:

```suggestion
/// my_actor_builder.connect_sink(NoConfig, &test_box_builder);
/// test_box_builder.connect_sink(NoConfig, &my_actor_builder);
```